src/Auth.php

```diff
@@ -221,7 +222,7 @@ public static function get_user_jwt_secret( $user_id ) {
 		 * If the request is not from the current_user or the current_user doesn't have the proper capabilities, don't return the secret
 		 */
 		$is_current_user = ( $user_id === get_current_user_id() ) ? true : false;
-		if ( ! $is_current_user || ! current_user_can( $capability ) ) {
+		if ( ! $is_current_user && ! current_user_can( $capability ) ) {
```

Comment:
I wonder why you put && here @jasonbahl . Shouldn't it make sure to return null if either of those conditions is true?